crates/notary/server/tee/notary-server-sgx.md

```diff
@@ -0,0 +1,68 @@
+# TLSNotary Notary Server for Intel SGX
+
+This package contains the **SGX-enabled version of the TLSNotary Notary Server**, built with **Gramine** to run securely on Intel SGX hardware. Follow the instructions below to install dependencies, verify the integrity of the enclave, and launch the server.
+
+## Install Intel SGX Drivers & Runtime Dependencies
+
+Ensure your system has the required Intel SGX components installed.
+
+```sh
+wget https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key
+cat intel-sgx-deb.key | sudo tee /etc/apt/keyrings/intel-sgx-keyring.asc > /dev/null
+
+# Add the repository to your sources:
+echo 'deb [signed-by=/etc/apt/keyrings/intel-sgx-keyring.asc arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu noble main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+
+sudo apt-get update
+sudo apt-get install libsgx-epid libsgx-quote-ex libsgx-dcap-ql -y
+```
+
+For more details, refer to the official **[Intel SGX Installation Guide](https://download.01.org/intel-sgx/latest/dcap-latest/linux/docs/Intel_SGX_SW_Installation_Guide_for_Linux.pdf).**
+
+## Verify the Enclave Signature (`MR_ENCLAVE`, `MR_SIGNER`)
+
+Before running the Notary Server, verify that the provided **enclave signature (`.sig`)** matches the expected **`MR_ENCLAVE` and `MR_SIGNER`** values.
+
+Run the following command inside a **Gramine Docker container** to inspect the enclave's signature:
+
+```sh
+docker run --rm -v "$(pwd):/work" -w /work gramineproject/gramine:latest \
+  "gramine-sgx-sigstruct-view --verbose --output-format=json notary-server.sig"
+```
+
+The output should be the same as `notary-server-sigstruct.json`
+
+## Run the Notary Server in SGX
+
+Once verification is complete, you can start the Notary Server inside a **Gramine-protected SGX enclave**.
+
+```sh
+docker run --detach \
+  --restart=unless-stopped \
+  --device=/dev/sgx_enclave \
+  --device=/dev/sgx_provision \
+  --volume=/var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
+  --publish=7047:7047 \
+  --volume="$(pwd):/work" \
+  --workdir=/work \
+  gramineproject/gramine:latest \
+  "gramine-sgx notary-server"
```

Comment:
Following the comment above (assuming it's correct), I was actually expecting this document to show people how to build and run notary-server in SGX entirely locally (e.g. via the dockerfile); or are you advocating for people to directly use the binary built in our GH instead?

Reply:
Yes. I'll document how to verify with a local build later. This md file is for inclusion in the zip file. I'll probably add a README with more general documentation later.

Reply:
@heeckhau cool gotcha, or can have a line in this doc to make it clear that this is for the zip file